test/test_client.py

```diff
@@ -230,15 +238,17 @@ def test_handshake_bad_extension():
     assert str(excinfo.value) == "unrecognized extension bad"
 
 
-def test_protocol_error():
+def test_protocol_error() -> None:
     client = WSConnection(CLIENT)
     client.send(Request(host="localhost", target="/"))
     with pytest.raises(RemoteProtocolError) as excinfo:
         client.receive_data(b"broken nonsense\r\n\r\n")
     assert str(excinfo.value) == "Bad HTTP message"
 
 
-def _make_handshake_rejection(status_code, body=None):
+def _make_handshake_rejection(
+    status_code: int, body: Optional[bytes] = None
+) -> List[Event]:
```

Comment:
should we always do one-argument-per-line if we do the multi-line function header?

Reply:
This is also now `black`'s choice, it will stick to one line if it can, then go for multi-line.